app/views/shares/index.html.erb

```diff
@@ -25,6 +35,7 @@
         </div>
       </div>
     <% end %>
+>>>>>>> master
```

Comment:
I think this line of code needs to be deleted because it was added as part of a merge conflict.